react/week2/todoapp/src/App.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react";
+import "./index.css";
+import ToDoList from './ToDoList'
+import Timer from './Timer.js'
+
+function App() {
+  const[inputlist,setInputList]=useState();
+  const[items,setItems]= useState([]);
+
+  const itemEvent=(event)=>{
+    setInputList(event.target.value);
+
+  }
+
+  const lisOfItems=()=>{
+    setItems((oldItem)=>{
+      return[...oldItem,inputlist];
```

Comment:
This is a bit confusing to me. oldItem is actually an array of several todo items, while inputList is an array of a single input. See you switched up the singular - plural form here.

The alternative for oldItem could be "oldTodos" or better "prevTodos", 
and alternative for "inputlist" could be input or userInput as I suggested above.

On a more positive note: this is a good usage of the "prevState" and the spread operator.